# Define the goal
float32 new_position
---
# Define the result
bool movement_done
---
# Define a feedback message
float32 current_position